src/components/ToolMenu/Draw/Attributions/index.tsx

```diff
@@ -0,0 +1,216 @@
+import React, {
+  useEffect, useState
+} from 'react';
+
+import {
+  MinusCircleOutlined, PlusOutlined
+} from '@ant-design/icons';
+import {
+  Button, Drawer, Form, Divider, Row
+} from 'antd';
+import _cloneDeep from 'lodash/cloneDeep';
+import OlFeature from 'ol/Feature';
+
+import { useTranslation } from 'react-i18next';
+
+import {
+  useMap
+} from '@terrestris/react-geo/dist/Hook/useMap';
+
+import './index.less';
+import AttributionRow from './AttributionRow';
+
+export type AttributionDrawerProps = {
+  openAttributeDrawer: boolean;
+  onClose: (open: boolean) => void;
+};
+
+const AttributionDrawer: React.FC<AttributionDrawerProps> = ({
+  openAttributeDrawer,
+  onClose
+}) => {
+  const [selectedFeature, setSelectedFeature] = useState<OlFeature>();
+  const [render, setRender] = useState<boolean>(true);
+  const [isFormValid, setIsFormIsValid] = useState(true);
+  const [currentProperties, setCurrentProperties] = useState<any>({});
+
+  const [form] = Form.useForm();
+
+  // ToDo: will be used in future once react-geo library triggeres selected events
+  // const selectedFeatures: OlFeature[] = useAppSelector(state => state.selectedFeatures);
+
+  const map = useMap();
+
+  const {
+    t
+  } = useTranslation();
+
+  useEffect(() => {
+    const properties = selectedFeature?.getProperties();
+
+    setCurrentProperties({ ...properties });
+
+    const fs: any = {};
+    if (properties) {
+      Object.entries(properties).forEach(([key, value]) => {
+        fs[key] = {
+          name: key,
+          value: value
+        };
+      });
+    }
+
+    form.setFieldValue('fields', {});
+    form.setFieldsValue({
+      fields: fs
+    });
+  }, [selectedFeature, form]);
+
+  // ToDo: will be removed in future once react-geo library triggeres selected events
+  const selectInteraction: any = map?.getInteractions().getArray().filter(interaction => {
+    if (interaction.get('active') === true && interaction.get('name') === 'react-geo-select-interaction') {
```

Comment:
What happens if there is more than a single select interaction available in the application?

Reply:
That is a good point and needs to be revisited. To ensure getting the correct interaction, the interactions name needs to be set in the application. Currently the name is set within react-geo and can not be modified. I think the best way fixing that issue is to revisit react-geo and pass the interaction's name as a property.

Reply:
Maybe we can reset the name of the appropriate interaction manually via `myInteraction.set('name', 'myInteractionName')`?

Reply:
That is a very good idea and did work out!

Reply:
A Todo is left behind.